index.html

```diff
@@ -1205,7 +1205,7 @@ <h6 data-lang="zh-hant">發表對本文檔的建議</h6>
                   <p data-lang="zh-hans">文字由左向右，行由上向下排列。</p>
                   <p data-lang="zh-hant">文字由左而右，行由上而下排列。</p>
                   <div class="note">
-                    <p data-lang="en">Traditional Chinese mainly uses vertical writing mode. When characters need to be composed horizontally, each character will be arranged in a single horizontal line from right to left, and no columns will be applied. Such a layout is considered a special case for vertical writing mode, often seen on stone inscriptions and headings for newspapers and magazines in the past. Now it has largely been replaced by the horizontal writing mode arrangement from left to right.</p>
+                    <p data-lang="en">Vertical writing mode is mainly used in Traditional Chinese. When characters need to be composed horizontally, each character will be arranged in a single horizontal line from right to left, and no columns will be applied. Such a layout is considered a special case for vertical writing mode, often seen on stone inscriptions and headings for newspapers and magazines in the past. Now it has largely been replaced by the horizontal writing mode arrangement from left to right.</p>
```

Comment:
I think the original Chinese text means "only" instead of "mainly" here, and my personal translation of the first sentence is: Chinese traditionally only uses vertical writing mode.

Reply:
Okay, that does make sense. Will make the change.